parallel-consumer-core/src/main/java/io/confluent/parallelconsumer/offsets/EncodedOffsetPair.java

```diff
@@ -114,6 +114,9 @@ public HighestOffsetAndIncompletes getDecodedIncompletes(long baseOffset) {
             case BitSetV2Compressed -> deserialiseBitSetWrapToIncompletes(BitSetV2, baseOffset, decompressZstd(data));
             case RunLengthV2 -> runLengthDecodeToIncompletes(encoding, baseOffset, data);
             case RunLengthV2Compressed -> runLengthDecodeToIncompletes(RunLengthV2, baseOffset, decompressZstd(data));
+            case KafkaStreams, KafkaStreamsV2 ->
+                    throw new KafkaStreamsEncodingNotSupported(
+                            "It looks like you're reusing a Kafka Streams consumer group id. This isn't supported. Please, use a fresh consumer group, unique to PC");
```

Comment:
move the message into the exception as a constant? seeing as there are no other uses of the exception, I think that would make it more understandable.
Also,
```suggestion
                            "It looks like you might be reusing a Kafka Streams consumer group id, as KS magic numbers were found in the serialised payload, instead of our own. Using PC on top of KS commit data isn't supported. Please, use a fresh consumer group, unique to PC.");
```